docs/src/tutorial/temporal_resolution.md

```diff
@@ -0,0 +1,175 @@
+# Temporal Resolution Tutorial
+
+Welcome to Spine Toolbox's Temporal Resolution tutorial.
+
+This tutorial provides a step-by-step guide to include uniform and/or flexible temporal resolution in your model.
+
+For more information on how time works in SpineOpt, see the [Temporal Framework](https://spine-tools.github.io/SpineOpt.jl/latest/advanced_concepts/temporal_framework/) documentation.
+
+## Introduction
+
+### Installation and upgrades
+
+If you haven't yet installed the tools yet, please follow the installation guides: 
+- For Spine Toolbox: [Spine Toolbox installation guide](https://github.com/spine-tools/SpineOpt.jl#installation)
+- For SpineOpt: [SpineOpt installation guide](https://github.com/spine-tools/Spine-Toolbox#installation)
+
+If you are not sure whether you have the latest version, please upgrade to ensure compatibility with this guide.
+
+- For Spine Toolbox:
+    - If installed with pipx, then use `python -m pipx upgrade spinetoolbox`
+    - If installed from sources using git, then `git pull`, `python -m pip install -U -r requirements.txt`
+
+- For SpineOpt: [SpineOpt upgrade guide](https://github.com/spine-tools/SpineOpt.jl#upgrading)
+
+### Model assumptions 
+
+- This tutorial builds upon the Simple System tutorial. Please follow [that tutorial](https://spine-tools.github.io/SpineOpt.jl/latest/tutorial/simple_system/) first, or download the [finished system](**NEEDS LINK**).
+- The scenario runs for 5 hours and the default temporal resolution is 1 hour.
+- The fuel_node has a temporal resolution of [1, 1, 1, 2] hours.
+
+## Tutorial
+
+### Adjustments to Simple System
+
+Before demonstrating flexible temporal resolution, we need to adjust the Simple System tutorial so the results are more interesting to examine.
+
+!!! note If you are ever missing a window in the Spine DB Editor, go to the *Hamburger menu* (top right) > under View: Docks > Choose the missing window - then drag and resize the window to your preference
+
+#### Adding variable electricity demand
+
+In SpineToolbox, double click on the *InputData*  block.
+
+In the *Object tree* window:
+- Expand "node"
+- Select the electricity node
+- In the *Object parameter value* window, find the value for the electricity node's demand (should be 150 from the Simple System) - right click > Edit\
+	`Parameter type: Time Series fixed resolution`\
+	Copy and paste (or fill in the table):
+	
+		10
+		20
+		40
+		50
+		20
+
```

Comment:
I don't know whether it matters but with this (low) demand you effectively disable unit b.

Reply:
Ha yeah. The graph is quite boring. Do you think it matters enough to change all the graphs?

Reply:
Not sure, there are advantages and disadvantages in doing so. Now it is simpler and therefore easier to follow. And it may be good that there are a few easy tutorials for confidence boosting. On the other hand, it could be interesting to compare two similar units with different resolutions (though that would require us to introduce another node which may introduce more errors).

Perhaps we give a hint that this is something that they can try to do themselves but which is not covered in the tutorial (except perhaps one graph with the solution such that they know what to aim for)?

Reply:
I'll leave it simple as it is. It's more about HOW to put in resolution, not what cases and why. They should know that! ;)